### PR TITLE
Screenshot update qgis_configuration

### LINKED ISSUE #4755: Request in QGIS (Add setting for format to show angular bearings to projects)
### Request for documentation
From pull request QGIS/qgis#33674
Author: @nyalldawson
QGIS version: 

**Add setting for format to show angular bearings to projects**

### PR Description:
The Settings - Options - Map Tools tab contains a new setting for controlling the default format to use for displaying angular bearings for newly created projects. Whenever a new project is created, it will
inherit this default settings.

The Project Properties dialog also has a new setting for the project-specific bearing format.

The intention is that whenever angular bearings are shown in QGIS, they will be formatted using the current project's bearing format settings. In this PR I've done this for the status bar pan direction message.

Peek 2020-01-09 11-08

### Commits tagged with [need-docs]

### LINKED ISSUE #6259: [layer tree][ui] Allow users to set a minimum/maximum legend size (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#39777
Author: @nirvn
QGIS version: 3.18

**[layer tree][ui] Allow users to set a minimum/maximum legend size**

## Description

This PR fixes a long standing UI/UX issue with the layer tree panel whereas legend icons for symbols using map units can end up taking an undesirably large amount of space in the tree widget.

The PR introduces a new pair of settings, a minimum legend size value and a maximum legend size value. Those are tweaked in the settings dialog's canvas and legend panel:
image

By default, we now max out symbol sized above 32mm by default. Users can set this to 0 to allow symbols to grow infinitely as it was before. But really, why? :wink: 

Here's an example, with symbols clipped to 25 mm:
image

Funded by SO!GIS
CC @andreasneumann

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by felixfeckler**
Updated Screenshots that showed outdated Interace.
Added some menu items, that were not documented yet.

Goal:

Ticket(s): Fixes #6259, fixes #4755

- [ ] Backport to LTR documentation is required

### CODE REVIEW

**Review by DelazJ — COMMENTED** [review]
Thanks @felixfeckler 
PS: Remember to link the issues you are addressing so that we can (potentially) close them.

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -525,6 +525,7 @@ These properties let you set:
     panel, e.g. when applying a categorized or rule-based renderer
     (see :ref:`vector_style_menu` for more information).
   * the :guilabel:`WMS getLegendGraphic Resolution`
+  * :guilabel:`Minimum` and :guilabel:`Maximum legend symbol size`
```
[the remark this answers is not in the document]
Comment (felixfeckler) [reply docs/user_manual/introduction/qgis_configuration.rst:528]: Yes but the feature was also backported to 3.16, at least as I understand it in qgis/QGIS#39777, also i made the screenshots in QGIS 3.16, so 3.18 features shouldnt be included

[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/user_manual/introduction/qgis_configuration.rst:528]: Ah! You are right. I did not know it was backported. So everything is good. 
> i made the screenshots in QGIS 3.16, so 3.18 features shouldnt be included

Indeed 😄

[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/user_manual/introduction/qgis_configuration.rst:528]: Any issue with approving this suggestion?

**Pull request merged**